app/src/actioncommands.cpp

```diff
@@ -234,6 +241,15 @@ Status ActionCommands::exportGif()
 Status ActionCommands::exportMovie(bool isGif)
 {
     FileType fileType = (isGif) ? FileType::GIF : FileType::MOVIE;
+
+    int clipCount = mEditor->sound()->soundClipCount();
+    if (fileType == FileType::MOVIE && clipCount >= MovieExporter::MAX_SOUND_FRAMES)
+    {
+        ErrorDialog errorDialog(tr("Something went wrong"), tr("You currently have a total of %1 sound clips. Due to current limitations, you will be unable to export any animation exceeding %2 sound clips. We recommend splitting up larger projects into multiple smaller project to stay within this limit.").arg(clipCount).arg(MovieExporter::MAX_SOUND_FRAMES), QString(), mParent);
```

Comment:
Any reason that not using `showSoundClipWarning()` here?

Reply:
The former is a warning dialog, the latter is an error, so I think it's fine